<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Widget" name="openGWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>771</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_file">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_collor_edje">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>250</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет ребер</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_edje">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ребра</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vertex">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>339</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вершины</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_collor_vertex">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>340</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет вершин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Отрисовка объекта</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>430</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Перемещение по осям</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>460</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ось X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>490</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ось Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>520</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ось Z</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_moving_X">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>460</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="C" country="AnyTerritory"/>
    </property>
    <property name="minimum">
     <number>-150</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_moving_Y">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>490</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-150</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_moving_Z">
    <property name="geometry">
     <rect>
      <x>969</x>
      <y>520</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-150</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>550</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вращение</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_rotation_X">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>610</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDial" name="dial_rotation_Y">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>610</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QDial" name="dial_rotation_Z">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>610</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_screenshot">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gif">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>gif</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>750</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштаб</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>740</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Проекция</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_type_edje">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>250</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Сплошная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пунктирная</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_type_vertex">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>340</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Круг</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Квадрат</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Отсутствует</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_projection">
    <property name="geometry">
     <rect>
      <x>902</x>
      <y>741</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Паралельная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Центральная</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>180</y>
      <width>58</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Фон</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_collor_background">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет фона</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_collor_background">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_collor_edje">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_collor_vertex">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>340</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: black;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_size_edje">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>290</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>290</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Толщина ребер</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>380</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер вершин</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_size_vertex">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>380</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>590</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ось X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>590</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ось Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>590</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ось Z</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_rotation_X">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>700</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_rotation_Y">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>700</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_rotation_Z">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>700</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>740</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_count_dot_text">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество точек: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count_poligons_text">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>80</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество полигонов: </string>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_count_dot">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count_poligons">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>80</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_horizontal_X">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>460</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-150</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_horizontal_Y">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>490</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-150</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_horizontal_Z">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>520</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-150</number>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1327</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
